mitosheet/mitosheet/transpiler/transpile.py

```diff
@@ -87,10 +88,18 @@ def transpile(
     # We then deduplicate the imports, keeping the same order as originally
     final_imports_code = deduplicate_array(imports_code)
 
+    # If the user wants the custom imports to be included in the script, we include them here
+    if steps_manager.code_options['import_custom_python_code']:
```

Comment:
Currently, we don't create the spreadsheet widget, and our tests are failing, if `code_options` is not none, but you don't set `import_custom_python_code`. 

I think since there are so many code options at this point, we should set default values for them and render the sheet. 

The most helpful default here seems like "If you are returning a function or the function code, then `import_custom_python_code` should be `True`.

Reply:
Passing this information down is a bit of a hassle, and gets a bit confusing with the default. Keeping the minor refactor of just defaulting to False everywhere for now.